app/build.gradle.kts

```diff
@@ -84,7 +84,7 @@ android {
 }
 
 dependencies {
-
+    implementation("androidx.datastore:datastore-preferences:1.1.1")
```

Comment:
this should be a reference to the libs.versions.toml file so that we keep all the dependencies in one place